src/journal/journald-server.c

```diff
@@ -560,24 +559,13 @@ static int server_archive_offline_user_journals(Server *s) {
                 if (!de) {
                         if (errno != 0)
                                 log_warning_errno(errno, "Failed to enumerate %s, ignoring: %m", s->system_storage.path);
-
                         break;
                 }
 
-                a = startswith(de->d_name, "user-");
-                if (!a)
-                        continue;
-                b = endswith(de->d_name, ".journal");
-                if (!b)
-                        continue;
-
-                u = strndup(a, b-a);
-                if (!u)
-                        return log_oom();
-
-                r = parse_uid(u, &uid);
+                r = journal_file_parse_uid(de->d_name, &uid, false);
```

Comment:
in new code we generally want that boolean function parameters (or in fact any parameters where the passed value gives you no hint at all what is passed here, for example NULL parameters) are prefixed with a comment such as `/* parse_archive= */`, to make this more readable. i.e.:

```c
r = journal_file_parse_uid(de->d_name, &uid, /* parse_archive= */ false);
```

please follow this syntax here too